### DISCUSSION TITLE
Github Page Share link on 404 Error

### DISCUSSION THREAD

**Discussion by MrShankarAryal** [community/community#139188] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My website [https://mrshankararyal.github.io](url) when i share on social media it throw 404 error. As I look when I directly search my website in Brower it is opening. The problem is when sharing the my website link  it gives 404 error when open. The hosted Code is in this respiratory  [https://github.com/[MrShankarAryal/mrshankararyal.github.io](https://github.com/MrShankarAryal/mrshankararyal.github.io)](url)

**Comment by leonlime** [comment]
Indeed, if I click in your link, don't work but if I write https://mrshankararyal.github.io on my browser, works perfectly.

**Reply by MrShankarAryal** [reply]
yes why this happening is there any solutions?

**Reply by leonlime** [reply]
I'm testing in my social media, works fine if i share your site on my teams or whatsapp

**Reply by MrShankarAryal** [reply]
it won't be working in some social media likes Instagram and GitHub this community also

**Reply by leonlime** [reply]
Through your website, I found your social links, the link is working fine! maybe the problem is with the browser or some cache.

**Reply by MrShankarAryal** [reply]
it is working and working on others app also but the problem arise when i send someone in chats or write in any comments or discussion like here

**Reply by MrShankarAryal** [reply]
> Through your website, I found your social links, the link is working fine! maybe the problem is with the browser or some cache.

i have try using different browser but doesn't work. Note that the problem is arising when i am  sending someone in **chat**s of social medias and other apps

**Reply by leonlime** [reply]
can you send the link to me in some social media?

instagram: leokaneda

**Reply by MrShankarAryal** [reply]
I just solve problem by adding this `<link rel="canonical" href="https://mrshankararyal.github.io/" />`

**Comment by leonlime** [comment, marked as answer]
solved adding a /index.html on the final of the link. like mrshankararyal.github.io/index.html
